### PR TITLE
Add option to start games in fullscreen mode

### LINKED ISSUE #1253: Feature  request - Add ability to launch in fullscreen from command prompt.
This would be very helpful when using the emulator in a frontend. Something like:
Ryujinx.exe gamename -f

### LINKED ISSUE #1565: Feature Request: Option to start games in fullscreen
#### Overview:
I would love to see an option to start games in fullscreen.

#### Smaller Details: 
Maybe add a tickbox under graphic settings saying "Start games in fullscreen".
A command line option would also be great.

### Why would this feature be useful?
This would be useful for people that would like to skip the step of pressing F11 to go to fullscreen after starting a game.
It would also simplify launching games  using a frontend. At the moment, you will need to use an external script that presses F11  for you or pull out your keyboard and press F11 if you want to play in fullscreen.

### PR DESCRIPTION
**Pull request by Xpl0itR**
Add option to start games in fullscreen mode.

Closes #1565 and closes #1253

### CODE REVIEW

File: Ryujinx/Ui/MainWindow.glade [hunk 1]
```diff
@@ -112,11 +112,26 @@
...
+                        <property name="visible">True</property>
+                        <property name="can_focus">False</property>
+                      </object>
+                    </child>
                     <child>
                       <object class="GtkMenuItem" id="_stopEmulation">
                         <property name="visible">True</property>
                         <property name="can_focus">False</property>
-                        <property name="tooltip_text" translatable="yes">Stop emualtion of the current game and return to game selection</property>
+                        <property name="tooltip_text" translatable="yes">Stop emulation of the current game and return to game selection</property>
```
Comment (Thog) [inline Ryujinx/Ui/MainWindow.glade:134]: nice catch 👍

File: Ryujinx.Common/Configuration/ConfigurationState.cs [hunk 2]
```diff
@@ -1,4 +1,4 @@
-﻿using Ryujinx.Common;
+using Ryujinx.Common;
```
Comment (Thog) [inline Ryujinx.Common/Configuration/ConfigurationState.cs:1]: weird empty change? maybe line return changed?

**Review by Thog — APPROVED** [review]
looks good but the line ending seems to be broken on some files..?

File: Ryujinx.Common/Configuration/ConfigurationState.cs (hunk 2 above)
Comment (Xpl0itR) [reply Ryujinx.Common/Configuration/ConfigurationState.cs:1]: VS says all the line endings are LF, so maybe that one was CRLF before and was automatically changed to LF to match the rest

**Comment by EmulationFanatic** [comment]
Works fine in my testing, though it should be noted that if you have already enabled the "Start games in fullscreen mode" menu option, there is no need to use the --fullscreen command line option thereafter.

Additionally, all other top-level menu items use camel case. My unsolicited suggestion would be to do the same here so that it reads "Start Games in Fullscreen Mode"

**Comment by Dime333** [comment]
Maby split this PR into two different PRs. One for the menu option and one for the command line?

**Review by riperiperi — APPROVED** [review]
Nice! This is a good feature to have.

**Comment by Thog** [comment]
Rebase required because of https://github.com/Ryujinx/Ryujinx/commit/aa129fdbdfd424f73da2b187fc28855f8ae90059.

**Review by AcK77 — APPROVED** [review]

**Pull request merged**